en/computer_vision/collision_prevention.md

```diff
@@ -40,6 +41,32 @@ Enabled collision prevention by [setting the following parameter](../advanced_co
 If you are using a distance sensor attached to your flight controller for collision prevention, it will need to be [attached and configured](#rangefinder) as described in the next section.
 If you are using a companion computer to provide obstacle information see [companion setup](#companion).
 
+### MPC_COL_PREV_ANG Angle Tuning {#angle_tuning}
+
+Collision prevention calculates the vehicle speed using the minimum distance from *the set of all included sensors* in the commanded direction, where the "included set" is tuned using [MPC_COL_PREV_ANG](../advanced_config/parameter_reference.md#MPC_COL_PREV_ANG).
```

Comment:
" Collision prevention calculates the allowed vehicle speed using a subset of the range scan provided by all sensors. The size of the this subset (centered around the commanded direction) can be tuned by the parameter MPC_COL_PREV_ANG."